cli/args/mod.rs

```diff
@@ -843,32 +871,15 @@ impl CliOptions {
     }
   }
 
-  pub async fn resolve_npm_resolution_snapshot(
+  pub fn resolve_npm_resolution_snapshot(
     &self,
-    api: &CliNpmRegistryApi,
   ) -> Result<Option<ValidSerializedNpmResolutionSnapshot>, AnyError> {
     if let Some(state) = &*NPM_PROCESS_STATE {
       // TODO(bartlomieju): remove this clone
-      return Ok(Some(state.snapshot.clone().into_valid()?));
-    }
-
-    if let Some(lockfile) = self.maybe_lockfile() {
-      if !lockfile.lock().overwrite {
-        let snapshot = snapshot_from_lockfile(lockfile.clone(), api)
-          .await
-          .with_context(|| {
-            format!(
-              "failed reading lockfile '{}'",
-              lockfile.lock().filename.display()
-            )
-          })?;
-        // clear the memory cache to reduce memory usage
-        api.clear_memory_cache();
-        return Ok(Some(snapshot));
-      }
```

Comment:
Has this been merged into somewhere else?

Reply:
Yeah, it's internal to the npm module.